plugins/blocks/subscription/locale/fa_IR/locale.xml

```diff
@@ -1,25 +1,23 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE locale SYSTEM "../../../../../lib/pkp/dtd/locale.dtd">
-
 <!--
   * plugins/blocks/subscription/locale/fa_IR/locale.xml
   *
-  * Copyright (c) 2013-2017 Simon Fraser University
-  * Copyright (c) 2003-2017 John Willinsky
+  * Copyright (c) 2013-2018 Simon Fraser University
+  * Copyright (c) 2003-2018 John Willinsky
   * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
   *
-  * Credits: https://pkp.sfu.ca/wiki/index.php?title=OJS:_Farsi_(fa_IR)
```

Comment:
Would you be willing to leave a notice like this in the headers, and then we could add credit in the wiki? We're trying to keep our code headers from accumulating too many names -- and we want to be fair when translations are adopted or work is shared.